sklearn/impute/tests/test_impute.py

```diff
@@ -1546,6 +1547,46 @@ def test_iterative_imputer_constant_fill_value():
     assert_array_equal(imputer.initial_imputer_.statistics_, fill_value)
 
 
+@pytest.mark.parametrize(
+    "missing_column, check_column, min_value, max_value",
+    [
+        (2, 3, 4, 5),
```

Comment:
Also if you can add a comment why the result should be the expected one it would be great. This is probably super easy for you right now because you have this fresh in your memory but imagine someone (maybe even you) in 6 months that hasn't looked at this code for a while and need to figure out what this test is trying to check.

Reply:
Thank you for pointing this out. I have improved the test readability by removing the parametrization as you suggested. I also added comments to help understand the test case better.